lib/typelizer/inline_type.rb

```diff
@@ -0,0 +1,22 @@
+module Typelizer
+  # InlineType is mainly used with Alba plugin to represent inline associations.
+  # `name `method` is the same interface as `Interface` class.
+  class InlineType
+    TEMPLATE = <<~ERB.strip
```

Comment:
Heredoc is used as a template. This can be a separate file, but it is short enough.